services/api/src/utils/search.js

```diff
@@ -0,0 +1,102 @@
+const Joi = require('@hapi/joi');
+const ObjectId = require('mongoose').Types.ObjectId;
+const PassThrough = require('stream').PassThrough;
+const csv = require('fast-csv');
+
+function searchValidation(options = {}) {
+  const { limit = 50, sortField = 'createdAt', sortOrder = 'desc' } = options;
+  return {
+    startAt: Joi.date(),
+    endAt: Joi.date(),
+    skip: Joi.number().default(0),
+    sort: Joi.object({
+      field: Joi.string().required(),
+      order: Joi.string().required(),
+    }).default({
+      field: sortField,
+      order: sortOrder,
+    }),
+    limit: Joi.number().positive().default(limit),
+    keywords: Joi.string(),
```

Comment:
this is a single keyword no? ie. not comma separated or anything? just curious why it's not just `keyword`

Reply:
should be an array i suspect

Reply:
It's a single `keyword` - you got me plummer!

Reply:
Fixed